xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/internal/job/AbstractEntityJobWithChecks.java

```diff
@@ -63,13 +64,28 @@ protected void runInternal() throws Exception
                 // Process
                 progressManager.startStep(this);
                 setContextUser();
+
+                // FIXME: this should probably be the selected entities only...
                 process(entityReferences);
             }
         } finally {
             progressManager.popLevelProgress(this);
         }
     }
 
+    /**
+     * @return the list of references that have been selected to be refactored.
+     * @since 16.10.0RC1
```

Comment:
Shouldn't this since-version be updated?

Reply:
Yes it should.